### PR TITLE
add script to format authors

### PR DESCRIPTION
**Pull request by rkaminsk**

### CODE REVIEW

**Comment by rkaminsk** [comment]
@jorgefandinno and @tortinator, would you please check if the changes to the authors are okay?

File: procs.bib [hunk 1]
```diff
@@ -1200,7 +1200,7 @@ @proceedings{damp10
 
 @proceedings{datalog10,
   title = {Proceedings of the First International Workshop on Datalog},
-  editor = {O. {de Moor} and G. Gottlob and T. Furche and A, Sellers},
+  editor = {O. {de Moor} and G. Gottlob and T. Furche and S. A},
```
Comment (jorgefandinno) [inline procs.bib:1203]: This should be "A. Sellers". It was wrongly introduced in the original file.

**Review by jorgefandinno — CHANGES_REQUESTED** [review]

File: procs.bib (hunk 1 above)
Comment (jorgefandinno) [inline procs.bib:1203]: editor = {O. {de Moor} and G. Gottlob and T. Furche and A. Sellers},

Comment (jorgefandinno) [reply procs.bib:1203]: This should be "A. Sellers". It was wrongly introduced in the original file.

Comment (rkaminsk) [reply procs.bib]: Thanks, merged your PR.

**Comment by rkaminsk** [comment]
I had another quick pass and the authors look fine to me now.

File: procs.bib (hunk 1 above)
Comment (jorgefandinno) [reply procs.bib]: The notes belong to a previous version. Should we describe the new version? Should we do it in a separated file or in the README.md?

**Comment by jorgefandinno** [comment]
I removed it from the description the functionality that is not in the script:
- It generates BibTeX keys from the author names and the year using the
  bibliography conventions
- It replaces journal names by their corresponding BibTeX strings
- It finds crossrefs in the procs.bib file and uses them for crossref field
- It flags repeated entries
This was part of a previous script that we discussed was too long to start with. If there is interest, we can later add this back.

**Comment by rkaminsk** [comment]
I made a pass over the authors and would merge now. The next step would be to include the functionality into the bibfmt.py script. We can however test the script for a while first.

**Comment by jorgefandinno** [comment]
> I made a pass over the authors and would merge now. The next step would be to include the functionality into the bibfmt.py script. We can however test the script for a while first.

Sounds good. Then, we can merge as it is.

**Pull request merged**